src/common_interface/integrator_types.jl

```diff
@@ -13,9 +13,10 @@ mutable struct DEOptions{SType,TstopType,CType}
     stop_at_next_tstop::Bool
 end
 
-abstract type AbstractSundialsIntegrator <: DiffEqBase.AbstractODEIntegrator end
+struct SundialsAlgorithm end
```

Comment:
why?

Reply:
I think I got confused. What is their type? Tell me and I can use it. I commented in a commit:

> I don't know what "use the actual alg" means. Which is the "actual" one?  `CVODE_Adams`? `CVODE_BDF`?

Reply:
Exactly. CVODE_Adams, CVODE_BDF, ARKODE, and IDA are the 4 algorithms here.

Reply:
`CVODEIntegrator` is one struct that has two algorithms. Should I just leave the type paramaeterized thenn?

Reply:
nah, it needs to pass the algorithm through and add it to the integrator.

Reply:
I am sorry Chris, I don't know how this works. I'll kindly ask you to add the finishing touches here, while I move on to making PRs to the other repos listed in the DiffEq issue.